src/views/OrderDetail.vue

```diff
@@ -0,0 +1,148 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-back-button default-href="home"></ion-back-button>
+      </ion-buttons>
+      <ion-title>{{ $t("Order details") }}</ion-title>
+    </ion-toolbar>
+  </ion-header>
+  <ion-content>
+    <ion-list>
+      <ion-item>
+        <ion-label>
+          <h2>Customer name</h2>
+          <p>Order ID</p>
+        </ion-label>
+        <ion-note slot="end">order time delta</ion-note>
+      </ion-item>
+    </ion-list>
+    <ion-item>
+      <ion-icon :icon="callOutline" slot="start"></ion-icon>
+      <ion-label>Phone Number</ion-label>
```

Comment:
If the text is not internationalized, then you can also remove it from en.json file.

Reply:
@Utkarshkaraiya Make all the static text internationalized.

Reply:
Done